android/build.gradle

```diff
@@ -5,18 +5,25 @@ buildscript {
   repositories {
     google()
     mavenCentral()
-    jcenter()
   }
 
   dependencies {
-    classpath 'com.android.tools.build:gradle:4.0.1'
+    classpath "com.android.tools.build:gradle:7.1.3"
     // noinspection DifferentKotlinGradleVersion
     classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
   }
 }
 
-apply plugin: 'com.android.library'
-apply plugin: 'kotlin-android'
+def isNewArchitectureEnabled() {
```

Comment:
What is the significance of `newArchEnabled`?

Reply:
Don't know, it's a react native thing that all new builds have

Reply:
https://reactnative.dev/blog/2022/03/15/an-update-on-the-new-architecture-rollout

Reply:
Ah, cool.